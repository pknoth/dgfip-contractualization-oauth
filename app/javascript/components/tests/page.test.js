import ReactTestRenderer from 'react-test-renderer'
import Page from '../page'

describe('components | Page', () => {
  describe('render', () => {
    const props = {
      children: '<p>Some html</p>'
    }
    const renderer = ReactTestRenderer.create(<Page {...props} />)
    it('should be defined', () => {
      expect(renderer).toBeDefined()
    })
    it('should match snapshot', () => {
      expect(renderer).toMatchSnapshot()
    })
  })
})
