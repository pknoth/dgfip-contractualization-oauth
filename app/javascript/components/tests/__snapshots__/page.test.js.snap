// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | Page render should match snapshot 1`] = `
Array [
  <header>
    <nav
      className="nav"
    >
      <div
        className="nav__container"
      >
        <a
          className="nav__link"
          href="/"
        >
          <img
            alt="Accueil de particulier.api.data.gouv.fr"
            className="nav__logo"
            src="/static/images/logo-api-particulier.png"
          />
        </a>
        <ul
          className="nav__links"
        >
          <li>
            <a
              href="/about"
              onClick={[Function]}
            >
              A propos
            </a>
          </li>
          <li>
            <a
              href="/documentation"
              onClick={[Function]}
            >
              Documentation technique
            </a>
          </li>
          <li>
            <a
              href="/contact"
              onClick={[Function]}
            >
              Contactez-nous
            </a>
          </li>
        </ul>
        <a
          className="nav__link"
          href="/"
        >
          <img
            alt="Ma sélection de jeux de données"
            className="nav__logo"
            src="../static/images/file-icon.svg"
          />
        </a>
      </div>
    </nav>
  </header>,
  <div>
    &lt;p&gt;Some html&lt;/p&gt;
  </div>,
  <footer
    className="footer"
  >
    <div
      className="container"
    >
      <div
        className="footer__logo"
      >
        <img
          alt="Logo Etalab"
          src="/static/images/etalab.svg"
        />
        <ul
          className="footer__social"
        >
          <li>
            <a
              href="https://twitter.com/etalab"
              title="Twitter"
            >
              <img
                alt="Twitter"
                className="icon icon-twitter"
                src="/static/images/social/twitter.svg"
              />
            </a>
          </li>
          <li>
            <a
              href="https://github.com/etalab"
              title="Github"
            >
              <img
                alt="Github"
                className="icon icon-github"
                src="/static/images/social/github.svg"
              />
            </a>
          </li>
          <li>
            <a
              href="https://www.facebook.com/etalab"
              title="Facebook"
            >
              <img
                alt="Facebook"
                className="icon icon-facebook"
                src="/static/images/social/facebook.svg"
              />
            </a>
          </li>
          <li>
            <a
              href="mailto:contact@particulier.api.gouv.fr"
              title="Nous écrire un mail"
            >
              <img
                alt="Email"
                className="icon icon-mail"
                src="/static/images/social/email.svg"
              />
            </a>
          </li>
        </ul>
      </div>
      <ul
        className="footer__links"
      >
        <li>
          <h2>
            particulier.api.gouv.fr
          </h2>
        </li>
        <li>
          <a
            href="https://www.data.gouv.fr/reference"
          >
            Données de référence
          </a>
        </li>
        <li>
          <a
            href="https://www.data.gouv.fr/terms"
          >
            Conditions générales d‘utilisation
          </a>
        </li>
        <li>
          <a
            href="https://www.data.gouv.fr/api"
          >
            API
          </a>
        </li>
      </ul>
    </div>
  </footer>,
]
`;
